src/document.rs

```diff
@@ -1,5 +1,30 @@
 use serde::{Deserialize, Serialize};
 
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct DocumentInsertOptions {
+    /// The name of the collection.
+    /// This is only for backward compatibility.
+    /// In ArangoDB versions < 3.0, the URL path was /_api/document and this query parameter was required.
```

Comment:
I don't think we should consider compatability back to arangodb below v3.0. 

If we really want this kind of compatability, then we should check the version of arangdb server and hide it behind. For user, the API should stay consistent.

Reply:
Yes ! I agree.